spec/miq_ae_playbook_method_spec.rb

```diff
@@ -33,8 +33,41 @@
     let(:obj)    { double("OBJ", :workspace => workspace) }
     let(:inputs) { { 'name' => 'Fred' } }
 
+    let(:mpr) do
+      FactoryGirl.create(:miq_provision_request, :requester => user)
+    end
+
+    let(:svc_mpr) do
+      MiqAeMethodService::MiqAeServiceMiqProvisionRequest.find(mpr.id)
+    end
+
+    let(:stpr) do
+      FactoryGirl.create(:service_template_provision_request, :requester => user)
+    end
+
+    let(:svc_stpr) do
+      MiqAeMethodService::MiqAeServiceServiceTemplateProvisionRequest.find(stpr.id)
+    end
+
+    let(:mpt) do
+      FactoryGirl.create(:miq_provision_task, :miq_request => mpr)
+    end
+
+    let(:svc_mpt) do
+      MiqAeMethodService::MiqAeServiceMiqProvisionTask.find(mpt.id)
+    end
+
+    let(:stpt) do
+      FactoryGirl.create(:service_template_provision_task, :miq_request => stpr)
+    end
+
+    let(:svc_stpt) do
+      MiqAeMethodService::MiqAeServiceServiceTemplateProvisionTask.find(stpt.id)
+    end
```

Comment:
Just curious why you decided to make these all multi-line blocks.  They do not exceed the line limit if done as a single line like the previous `let` blocks.